### ISSUE TITLE
--export-globals exporting in wrong format

### ISSUE THREAD

**Issue by jangchoe** [postmanlabs/newman#594]
This is possibly related to [573](https://github.com/postmanlabs/newman/issues/573).  

When using the `--export-globals` option, it seems like newman is exporting the global files as environment files instead of global files. Currently, it exports them like this: 

```javascript
{
  "values": [
    {
      "key": "user-email",
      "name": "user-email",
      "value": "my-email@mailinator.com",
      "type": "string"
    }
    // and other values
   ]
}
```
That is an postman environment file format. Globals are simply an array of objects. Like so:
```javascript
[
    {
      "key": "user-email",
      "name": "user-email",
      "value": "my-email@mailinator.com",
      "type": "string"
    }
    // and other values
]
```

**Comment by kunagpal** [comment]
@jangchoe The globals file format was an array of objects, and has now been changed to closely resemble the environment export format, leading to overall uniformity in variable handling operations. All exports are intended to use the object format, with a `values` key and other metadata present. The latest version of the Postman client uses this format too.

**Comment by jangchoe** [comment]
Ah it seems to be true. I just updated my postman and checked the globals. However, unless I'm mistaken, it seems like the `--globals` flag is still taking the old format. I think that was my original problem when I couldn't read the globals files that was exported by `--export-globals`.

**Comment by kunagpal** [comment]
@jangchoe We're working to release a proper fix for the same, you can track it's progress [here](https://github.com/postmanlabs/newman/pull/579)

**Comment by jangchoe** [comment]
I'll close and call it a duplicate.

**Comment by kunagpal** [comment]
@jangchoe Well, #579 is a pull request, and this is an issue. They are related, no doubt, but not duplicates of each other.

**Comment by kunagpal** [comment]
This has been fixed in https://github.com/postmanlabs/newman/pull/697, and shall be rolled out in a future Newman release.

**Comment by kunagpal** [comment]
This issue has been fixed in Newman `v3.2.0`, do give it a spin!

**Comment by jangchoe** [comment]
Looks great. It seems to work.
